rules.js

```diff
@@ -6,26 +6,30 @@ class HtmlTitle extends Rule {
       description: "Title of the page must be defined"
     };
   }
+  validateTitle(title) {
```

Comment:
If we're going with this approach, I'd prefer putting all the logic in the `validateTitle` function.

```javascript
validateTitle(event) {
//...
}
```

And below we would have:

```javascript
this.on("dom:ready", validateTitle)
```

Reply:
Thanks for your review. I will write the entire logic in the validateTitle function

Reply:
I'm not seeing this change